src/App.js

```diff
@@ -1,10 +1,46 @@
-import React from 'react';
+import React, { Component } from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import './App.css';
+import Home from './components/Home';
 import Calculator from './components/Calculator';
+import Quote from './components/Quote';
+import NavBar from './components/NavBar';
 
-const App = () => (
-  <div>
-    <Calculator />
-  </div>
-);
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <>
+        <Router>
+          <NavBar />
+          <section className="app-content">
+            <Routes>
+              <Route
+                path="/math-magicians"
+                element={<Home />}
```

Comment:
- [ ] <b>OPTIONAL:</b> Just to make sure that the home is your default route, meaning it will be the first page when the app is visited, instead of setting your path to `path="/math-magicians"`, set it to `path="/"`.